packages/config/src/arbitrum.json

```diff
@@ -26,7 +26,7 @@
   "questing_address": "0xDA3caD5e4F40062CECa6c1B979766BC0BAed8e33",
   "questing_start_block": 2416540,
   "randomizer_address": "0x8e79c8607a28fe1EC3527991C89F1d9E36D1bAd9",
-  "randomizer_start_block": 2416540,
+  "randomizer_start_block": 4955965,
```

Comment:
Maybe you want a separate one for smolverse, but we need this one as is for Bridgeworld.

Reply:
So I've actually just set it to the contract creation block: https://arbiscan.io/txs?a=0x8e79c8607a28fe1EC3527991C89F1d9E36D1bAd9&p=93

But this is a good point... I don't want to sync that far back anyway, so I'll create a new config entry that matches the Smol's start block

Reply:
Pushed that change

Reply:
Okay thanks. Looks like all of the bridgeworld contracts had the same ones anyways, copy/pasted